tally_ho/apps/tally/views/reports/administrative_areas_reports.py

```diff
@@ -3172,9 +3172,9 @@ def render_column(self, row, column):
         elif column == 'race_type':
             return str('<td class="center">'
                        f'{row["race_type"].name}</td>')
-        elif column == 'voting_district':
+        elif column == 'sub_con_code':
             return str('<td class="center">'
-                       f'{row["voting_district"]}</td>')
+                       f'{row["sub_con_code"]}</td>')
         elif column == 'order':
             return str('<td class="center">'
                        f'{row["order"]}</td>')
```

Comment:
Not related to this pr, but since it got touched we might want to refactor the `render_column` logic to be less verbose. Check peter-m's pr [here](https://github.com/onaio/tally-ho/blob/41115c3cd361d3080d484c907a3dfdbf2543141a/tally_ho/apps/tally/views/super_admin.py#L693).